### PR TITLE
style(art-quiz): translate and add newline to tooltip text

### PR DESCRIPTION
**Pull request by laurabeth**
The type of this PR is: **style**

This PR solves [GRO-1539]

### Description

A small change requested by Lois.

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[GRO-1539]: https://artsyproduct.atlassian.net/browse/GRO-1539?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx [hunk 1]
```diff
@@ -220,6 +222,9 @@ export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
     return <ArtQuizResultsLoader onReady={handleReady} />
   }
 
+  // Using split to create a deliberate break in text for better readability
+  const tooltipText = t("artQuizPage.artworksScreen.tooltip").split("\n")
```
Comment (dzucconi) [inline src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx:226]: This is pretty unusual. Do we have no support for multi-line text in the translations? (That seems like it'd be a pretty big oversight)

File: src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx [hunk 2]
```diff
@@ -298,7 +303,9 @@ export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
         </Flex>
 
         <Tooltip
-          content="Like it? Hit the heart. Not for you? Choose X."
+          content={tooltipText.map(text => (
+            <Text>{text}</Text>
```
Comment (dzucconi) [inline src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx:307]: You wouldn't want to alter the styling of the tooltip; it's currently `variant="xs"` though why not just adjust the width of the tooltip?

File: src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx (hunk 1 above)
Comment (laurabeth) [reply src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx:226]: It was actually the tooltip that was causing the issue. You're right about adjusting the tooltip width though. I'll try that instead.

File: src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx (hunk 2 above)
Comment (laurabeth) [reply src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx]: I realized that there's an issue adjusting the width of the tooltip. The bottom line `Not for you? Choose X.` is longer than the top. Splitting into 2 elements avoids this.

Comment (laurabeth) [reply src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx]: added correct styling

**Review by dzucconi — DISMISSED** [review]

**Review by TMcMeans — APPROVED** [review]
once the CircleCI tests pass again, can merge!

**Pull request merged**